internal/cli/cli.go

```diff
@@ -97,21 +96,14 @@ func preCommand(ctx context.Context, cmdName model.TiltSubcommand) (context.Cont
 	l := logger.NewLogger(logLevel(verbose, debug), os.Stdout)
 	ctx = logger.WithLogger(ctx, l)
 
-	ctx, cleanup, err := initMetrics(ctx, cmdName)
-	if err != nil {
-		l.Errorf("Fatal error initializing metrics: %v", err)
-		os.Exit(1)
-	}
-
-	stats.Record(ctx, CommandCountMeasure.M(1))
-
 	a, err := wireAnalytics(l, cmdName)
 	if err != nil {
 		l.Errorf("Fatal error initializing analytics: %v", err)
 		os.Exit(1)
 	}
 
 	ctx = tiltanalytics.WithAnalytics(ctx, a)
+	cleanup := func() error { return nil }
```

Comment:
it looks like `cleanup` was only used for metrics and we can just get rid of the return value

Reply:
done